src/surfing_penguin/API/survey_api.py

```diff
@@ -118,14 +139,33 @@ def post(self):
         return survey_functions.name_get_survey(api.payload['name'])
 
 
+@api.route('/search_survey_by_author')
+class search_survey_by_author(Resource):
+    """
+    Show the information of a survey, given its name or ID.
```

Comment:
Does this function search a survey or a list of surveys? If it is a list, could you add an s for all `survey` here?

Reply:
fix at next commit